workers/data_refinery_workers/processors/illumina.py

```diff
@@ -319,7 +317,7 @@ def _run_illumina(job_context: Dict) -> Dict:
     Expects a job_context which has been pre-populated with inputs, outputs
     and the column identifiers which the R script needs for processing.
     """
-    input_file_path = job_context["input_file_path"]
+    job_context["input_file_path"]
```

Comment:
It seems like we have a bug here or dead code.

Reply:
dead code, will remove the whole line